dbcon/joblist/tupleaggregatestep.cpp

```diff
@@ -366,16 +366,18 @@ TupleAggregateStep::TupleAggregateStep(
 {
     fRowGroupData.reinit(fRowGroupOut);
     fRowGroupOut.setData(&fRowGroupData);
+
+    fNumOfThreads = fRm->aggNumThreads();
+    fNumOfBuckets = fRm->aggNumBuckets();
+    fNumOfRowGroups = fRm->aggNumRowGroups();
+
     fAggregator->setInputOutput(fRowGroupIn, &fRowGroupOut);
 
     // decide if this needs to be multi-threaded
     RowAggregationDistinct* multiAgg = dynamic_cast<RowAggregationDistinct*>(fAggregator.get());
     fIsMultiThread = (multiAgg || fAggregator->aggMapKeyLength() > 0);
 
     // initialize multi-thread variables
```

Comment:
Take a comment with the asignments you moved.

Reply:
This is accidentally left over from the experiments, so I'll rollback it.